control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -3181,6 +3177,17 @@ components:
       required:
         - spec
         - state
+    VolumeSnapshots:
+      description: |-
+        Array of volume snapshots plus the next token for subsequent get requests when using pagination.
```

Comment:
`plus the next token` let's remove this for now and add this text when we add pagination.